01week/rockPaperScissors.js

```diff
@@ -7,10 +7,56 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+// an easier way to do this would have been if (hand1 === hand2) {
+//return 'its a tie!';
+//}
 
 function rockPaperScissors(hand1, hand2) {
```

Comment:
This pull request has two files in it For now we're only working on one file at a time. Please make sure your PR have only one file.